sda-commons-web-autoconfigure/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
Feature flag to enable mocking of final classes

Reply:
Is it because of `RequestAuthorizationContext`? It has a public constructor, so you may just mock the `HttpServletRequest` and create a real instance.